### DISCUSSION TITLE
Won't transcode to WAV

### DISCUSSION THREAD

**Discussion by peardox** [FactoryXCode/MfPack#82] [Announcements]
Changed to using Samples\\MfTranscode\\MfTranscode sample 3 (same issue in sample 2). This will only affect Delphi >= 10.1 as it's in an IFDEF

In dlgAudioFormats.pas you reference sgResolutions, this actually needs to be sgAudioFormats

Anyway, having made these changes and opening a re-named m4b (m4a is OK so used that) I now need to convert it so a 16Khz WAV. Why a WAV? Well it's going to be input for [whisper.cpp](https://github.com/ggerganov/whisper.cpp)

The problem is that it won't then transcode to a WAV. GetWinAudioEncoderFormats calls MFTranscodeGetAudioOutputAvailableTypes which says nope...

And WAV is definitely a supported type (#80)

**Comment by FactoryXCode** [comment]
Hi,

Well, WAV is supported, but maybe not the format you want to use?
I can't give you a straight answer on this, because I don't know your sourcecode. You can push your sourcecode to [the sand box](https://github.com/FactoryXCode/MfPack/tree/Master/MfPack/SandBox)
Regards Tony.

**Comment by FactoryXCode** [comment]
Hi, the support of the different formats of wav depends on the soundcard manufacturer. Not the specs of Windows. Regards Tony.
